feature-group-definitions/fetch.yml

```diff
@@ -0,0 +1,89 @@
+spec: https://fetch.spec.whatwg.org/
+caniuse: fetch
+status:
+  baseline: false
+  support:
+    chrome: "105"
+    chrome_android: "105"
+    edge: "105"
+compat_features:
+  - api.Headers
+  - api.Headers.Headers
+  - api.Headers.append
+  - api.Headers.delete
+  - api.Headers.entries
+  - api.Headers.forEach
+  - api.Headers.get
+  # - api.Headers.getSetCookie # later addition
+  - api.Headers.has
+  - api.Headers.keys
+  # - api.Headers.lexicographical_sorting # unsupported in Safari, for a long time
+  - api.Headers.set
+  - api.Headers.values
+  - api.Headers.@@iterator
+  - api.Request
+  - api.Request.Request
+  - api.Request.Request.cross_origin_stripped
+  # - api.Request.Request.init_attributionReporting_parameter # related to a newer API
+  # - api.Request.Request.init_browsingTopics_parameter # related to a newer API
+  # - api.Request.Request.init_priority_parameter # related to a newer API
+  - api.Request.Request.init_referrer_parameter
+  # - api.Request.Request.request_body_readablestream # later addition
+  - api.Request.Request.response_body_readablestream
+  - api.Request.arrayBuffer
+  - api.Request.blob
+  - api.Request.body
+  - api.Request.bodyUsed
+  - api.Request.cache
+  # - api.Request.cache.only_if_cached # Not widely implemented
```

Comment:
Same comment as above.

Reply:
I had not noticed, but I see that BCD actually flags that feature as experimental:
https://github.com/mdn/browser-compat-data/blob/main/api/Request.json#L586C34-L586C34

It would be interesting to gather the rationale that was used in BCD to set that flag.